doc/builders/fetchers.chapter.md

```diff
@@ -1,12 +1,39 @@
 # Fetchers {#chap-pkgs-fetchers}
 
-When using Nix, you will frequently need to download source code and other files from the internet. For this purpose, Nix provides the [_fixed output derivation_](https://nixos.org/manual/nix/stable/#fixed-output-drvs) feature and Nixpkgs provides various functions that implement the actual fetching from various protocols and services.
+Nixpkgs provides fetchers for different protocols and services. Fetcher's are functions that simplify downloading files.
```

Comment:
This is the introduction of the fetchers chapter. While the previous intro certainly wasn't great, you've removed essential information from it. It should answer succinctly:

- What is a fetcher?
- What is the purpose of a fetcher?

Furthermore, please fix a pervasive spelling mistake.

```diff
-fetcher's
+fetchers
```

"fetcher's" has a possessive meaning. Example: a fetcher's hash makes sure [...]
The plural is just "fetchers".

Reply:
Good catch on the extra apostrophes. Updated.

> Fetchers are functions that simplify downloading files.

What is missing from this? Is it too simplistic?